lib/auth/default_current_user_provider.rb

```diff
@@ -258,6 +258,7 @@ def lookup_user_api_user_and_update_key(user_api_key, client_id)
         raise Discourse::InvalidAccess
       end
 
+      api_key.update_columns(last_used_at: Time.now)
```

Comment:
We should test this as well in an integration test

Reply:
@nbianca This is a good read about dealing with Time zones in Rails http://danilenko.org/2012/7/6/rails_timezones/. The TLDR is that you almost always want to use `Time.zone.now` over `Time.now`